<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsPanel</class>
 <widget class="QWidget" name="GraphicsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>511</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="PushButton_Basic" name="pGraphicPanelBack">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image: url(&quot;:/images/main/graphic/fanhui@2x.png&quot;);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_graphic">
        <property name="styleSheet">
         <string notr="true">QLabel{
color: rgb(68, 68, 68);
}</string>
        </property>
        <property name="text">
         <string>图形</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_line_5">
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color: rgb(224, 224, 224);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2d">
      <property name="styleSheet">
       <string notr="true">QLabel{
color: rgb(102,102,102);
}</string>
      </property>
      <property name="text">
       <string>二维图形</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="5">
       <widget class="PushButton_2" name="pGraphic6">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="PushButton_2" name="pGraphic7">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="PushButton_2" name="pGraphic14">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PushButton_2" name="pGraphic9">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="PushButton_2" name="pGraphic1">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="PushButton_2" name="pGraphic8">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="PushButton_2" name="pGraphic2">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="PushButton_2" name="pGraphic13">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="PushButton_2" name="pGraphic12">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="PushButton_2" name="pGraphic4">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="PushButton_2" name="pGraphic5">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="PushButton_2" name="pGraphic11">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="PushButton_2" name="pGraphic3">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="PushButton_2" name="pGraphic10">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="PushButton_2" name="pGraphic15">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="PushButton_2" name="pGraphic16">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="PushButton_2" name="pGraphic18">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="PushButton_2" name="pGraphic19">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="PushButton_2" name="pGraphic17">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_line_6">
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color: rgb(224, 224, 224);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3d">
      <property name="styleSheet">
       <string notr="true">QLabel{
color: rgb(102,102,102);
}</string>
      </property>
      <property name="text">
       <string>三维图形</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="PushButton_2" name="pGraphic3d1">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="PushButton_2" name="pGraphic3d2">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="PushButton_2" name="pGraphic3d5">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="PushButton_2" name="pGraphic3d3">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_null">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="PushButton_2" name="pGraphic3d4">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_null_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_18">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton_2</class>
   <extends>QPushButton</extends>
   <header>Custom/PushButton/pushbutton_2.h</header>
  </customwidget>
  <customwidget>
   <class>PushButton_Basic</class>
   <extends>QPushButton</extends>
   <header>Custom/PushButton/pushbutton_basic.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
